src/main/java/com/elastic/support/util/RemoteSystem.java

```diff
@@ -185,6 +185,51 @@ public void copyLogs(List<String> entries, String logDir, String targetDir) {
         }
     }
 
+    /**
+    * On this function we will try to collect the journalctl logs, 
+    * Some services as Kibana installed with the RPM package will give the access to the logs using the journalctl command
+    * We will create a temperory file to copy those logs, then move them to the calling host.
+    *
+    * @param  String serviceName
+    * @param  String targetDir
+    *
+    * @return         void
+    */
+    @Override
+    public void copyLogsFromJournalctl(String serviceName, String targetDir) {
+
+        ChannelSftp channelSftp = null;
+        String tempDir = "templogs";
+        String mkdir = "mkdir templogs";
+        String journalctl = " journalctl -u {{SERVICE}} > '{{TEMP}}/{{SERVICE}}.log'";
```

Comment:
It's odd seeing structural differences between clearly copy/pasted code (`LocalSystem:: copyLogsFromJournalctl`). These string replacements won't trigger errors, unless the parameters are not trustworthy and are for-sure non-`null`.